src/test/java/com/wikia/webdriver/common/core/networktrafficinterceptor/NetworkTrafficInterceptor.java

```diff
@@ -55,6 +55,16 @@ public boolean searchRequestUrlInHar(String needle) {
     return false;
   }
 
+  public HarEntry getMatchingEntryByUrlPatternFromHar(String pattern) {
```

Comment:
Could you remove `searchRequestUrlInHar`, `checkAssetsStatuses` and `getResponseHttpStatusCode` and create one new `getMatchingEntryByUrlContainsFromHar` method which will be in consistency with `getMatchingEntryByUrlPatternFromHar` method?